tree/ntuple/v7/src/RNTupleParallelWriter.cxx

```diff
@@ -105,11 +105,6 @@ class RPageSynchronizingSink : public RPageSink {
       throw RException(R__FAIL("should never commit dataset via RPageSynchronizingSink"));
    }
 
-   RPage ReservePage(ColumnHandle_t columnHandle, std::size_t nElements) final
-   {
-      return fInnerSink->ReservePage(columnHandle, nElements);
-   }
-
```

Comment:
This means each fill context will now allocate its own pages instead of going through the shared, inner page sink. This is likely fine for the moment.

Reply:
True, actually intentionally. Eventually, every fill context should manage the memory budget independently.